### ISSUE TITLE
UX: provide default value for `-seed (bool)` option in help text

### ISSUE THREAD

**Issue by pataquets** [anacrolix/confluence#39]
As it's being done for `-tcpPeers` or `-utpPeers`, for instance.
I've read the code, from which I assume default is `false`, but found no `Default: ` example help text for sending a PR. I guess the command line parser lib is introspecting the options struct.

**Comment by anacrolix** [comment]
You are correct, the decision is made here: https://github.com/anacrolix/tagflag/blob/45431b4b2c93f05f0f6c6fff266a7817d35c8f69/usage.go#L82. I could make it more selective for when to show the defaults (always show for non-nil pointers, and everything else would probably be better). A further improvement would be to show the current value as parsed (I think that's actually what it's showing now, which isn't great). My current preference for arg parsing is what I'm working in https://github.com/anacrolix/bargle/tree/ng, but I don't want to subject the project to the churn to change to that, tagflag is pretty simple.
